rcljava/src/main/java/org/ros2/rcljava/RCLJava.java

```diff
@@ -246,12 +247,12 @@ public static Node createNode(final String nodeName) {
    *     structure.
    */
   public static Node createNode(final String nodeName, final String namespace, final Context context) {
-    return createNode(nodeName, namespace, context, false);
+    return createNode(nodeName, namespace, context, true, true, new ArrayList<String>(), false);
   }
 
-  public static Node createNode(final String nodeName, final String namespace, final Context context, final boolean allowUndeclaredParameters) {
-    long nodeHandle = nativeCreateNodeHandle(nodeName, namespace, context.getHandle());
-    Node node = new NodeImpl(nodeHandle, nodeName, context, allowUndeclaredParameters);
+  public static Node createNode(final String nodeName, final String namespace, final Context context, final boolean useGlobalArguments, final boolean enableRosout, final ArrayList<String> cliArgs, final boolean allowUndeclaredParameters) {
```

Comment:
Wouldn't be better to define a `NodeOptions` class for `rcljava`?

I think it's fine that the `nativeCreateNodeHandle` has a complicated list of arguments, but the user facing API should be nicer IMO.

Reply:
Not a bad idea.  I'll take another look at it.

Reply:
I ended up doing this in 6236aa8 .  It's much cleaner, thanks for the idea.